src/NuGet.Core/NuGet.ProjectModel/DependencyGraphSpec.cs

```diff
@@ -254,9 +246,61 @@ public static DependencyGraphSpec Union(IEnumerable<DependencyGraphSpec> dgSpecs
 
         public static DependencyGraphSpec Load(string path)
         {
-            var json = ReadJson(path);
+            using (var stream = new FileStream(path, FileMode.Open, FileAccess.Read, FileShare.Read))
+            using (var streamReader = new StreamReader(stream))
+            using (var jsonReader = new JsonTextReader(streamReader))
+            {
+                var dgspec = new DependencyGraphSpec();
+                int? format = null;
+                bool wasObjectRead;
+
+                try
+                {
+                    wasObjectRead = jsonReader.ReadObject(propertyName =>
+                    {
+                        switch (propertyName)
+                        {
+                            case "format":
+                                format = jsonReader.ReadAsInt32();
```

Comment:
Is the format used anywhere?

Reply:
Ha ha, no.  It wasn't even read in the old reader.  Silly that we created a format property and never read it.  I'll remove this.